tests/resources/expectations/models.test_llama.TestLlama2.test_model_output

```diff
@@ -1 +1 @@
-7.249414920806885e-06,0.0,0.009292446076869965,0.011145830154418945,0.003724828362464905,0.0067032501101493835,0.02313563972711563,0.031505607068538666,0.014552794396877289,0.00025850534439086914,0.034480541944503784,0.008191846311092377,0.02060232311487198,0.020930953323841095,0.020266838371753693,0.010351285338401794
\ No newline at end of file
+7.256865501403809e-06,0.0,0.00929248332977295,0.01114596426486969,0.0037247538566589355,0.006703168153762817,0.02313566952943802,0.03150583803653717,0.01455271989107132,0.0002584382891654968,0.03448055684566498,0.008191876113414764,0.020602405071258545,0.02093110978603363,0.020266816020011902,0.010351307690143585
```

Comment:
these are just rounding errors causing us to regenerate the expected values? is our tolerance too tight or it's really expected that fusing significantly changes the results? do the fused results have the same expected overall precision and this is just some order of operations difference?

Reply:
We expect the results to change as the weight names have changed and are now sorted in a different way which will produce different numbers

Reply:
or maybe we just re-generated everything because of a failure in model_weight_keys and this was within precision? if that's the case, maybe we should just regenerate the expected outputs on the specific tests that actually failed?

Reply:
For this particular test they don't seem to be out of order, it looks like every value is identical at least up to the first few significant digits

Reply:
any thoughts on this comment? this isn't the one where the order changed, we just have small rounding error differences on the values for the test_model_output tests. If they're within tolerance I think we should be able to regenerate the other expects without regenerating this one?

Reply:
I'll try removing this to see if it fails. Possible we have a strict tolerance.

Reply:
It is not necessarily about the order changing but we now have one tensor to represent 3 tensors (fused qkv). Currently we are initializing in the test based on:

```python
        torch.random.manual_seed(5)
        sd = uninitialized_model.state_dict()
        params = sorted(sd.keys())
        for key in params:
            parameter = sd[key]
            values = torch.randn_like(parameter)
            values -= 0.5
            values /= 20.0
            parameter.copy_(values)
        return uninitialized_model
```

Initializing the 3 tensors will give a different output than initializing the one tensor.

Reply:
My hypothesis here has two parts:
1. We are generating the same amount of random numbers, and they are probably getting generated in the same order (by pure chance), which reduces the error resulting from the random distribution.
2. We expect rounding errors to happen when fusing, as matrix multiplication is never fully exact either on GPU or CPU, which shows in these 1e-6/1e-7 errors we're seeing